readme.md

```diff
@@ -42,6 +49,6 @@ To run javascript outside the electron runtime, but with the same ECMAscript fea
 
 ## Client logfile locations:
 
-Windows: "C:\Users\dmf\AppData\Roaming\invest-workbench\"
-Mac: "~/Library/Application Support/invest-workbench/"
-Linux: "~/.config/invest-workbench/"
\ No newline at end of file
+Windows: "C:\Users\dmf\AppData\Roaming\invest-workbench\"  
+Mac: "\~/Library/Application Support/invest-workbench/"  
+Linux: "\~/.config/invest-workbench/"  

```

Comment:
Looks like some whitespace snuck in!

Reply:
😮 
two spaces is a newline in markdown-speak ;)

Reply:
Ha!